noir/Earthfile

```diff
@@ -81,6 +81,46 @@ build:
   BUILD +nargo
   BUILD +packages
 
+build-acir-tests:
+  FROM +nargo
+  ENV PATH="/usr/src/noir-repo/target/release:${PATH}"
+  WORKDIR /usr/src/noir-repo/test_programs
+  COPY . .
+  RUN ./rebuild.sh
+  SAVE ARTIFACT /usr/src/noir-repo/test_programs/acir_artifacts/*
+
+# This is ripped from end-to-end's Earthfile, need to pull from there directly.
+UPLOAD_LOGS:
```

Comment:
@spalladino wdyt about moving this function into the scripts Earthfile?

Reply:
I'm fine with it!